packages/ui/src/notification/NotificationBox.tsx

```diff
@@ -0,0 +1,15 @@
+import cn from 'classnames'
+
+type NotificationBoxProps = {
+  text: string
+} & React.HTMLAttributes<HTMLDivElement>
+
+const NotificationBox = ({ text, children, className, ...rest }: NotificationBoxProps) => {
+  return (
+    <div className={cn('caption-3A text-red-warning w-full max-w-md rounded-[20px] bg-white px-[20px] py-[15px]', className)} {...rest}>
```

Comment:
text 길이에 상관 없이 개행을 넣고싶을 때를 대비해서
`whitespace-pre-wrap` 요 클래스를 추가해주는건 어떨까요?!
```suggestion
    <div className={cn('caption-3A text-red-warning w-full max-w-md whitespace-pre-wrap rounded-[20px] bg-white px-[20px] py-[15px]', className)} {...rest}>
```